.pre-commit-config.yaml

```diff
@@ -136,3 +136,10 @@ repos:
         entry: python scripts/no_bool_in_generic.py
         language: python
         files: ^pandas/core/generic\.py$
+    -   id: always-use-future-annotations
+        name: ensure that "from __future__ import annotations" is always used
+        types: [python]
+        language: pygrep
+        args: [--negate]
+        entry: 'from __future__ import annotations'
+        exclude: __init__.py|api.py|setup.py|asv_bench|tests|web
```

Comment:
can you make this multi-line? See some of the other hooks in this file for examples

Reply:
ah right, i should have added those files too, was just going by the previous list. 

it's multiline now, which i had avoided at first, as (i think) it's a more complicated syntax for not much more readability. 

tested and working.